phabricatoremails/model/common.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from datetime import timezone
+from enum import Enum
+from typing import Optional
+
+
+@dataclass
+class Recipient:
+    email: str
+    timezone: timezone
+    is_actor: bool
+
+
+class ReviewerStatus(Enum):
+    ACCEPTED = "accepted"
+    REQUESTED_CHANGES = "requested-changes"
+    BLOCKING = "blocking"
+    UNREVIEWED = "unreviewed"
```

Comment:
Do the values here matter? If not, I would suggested using `auto` instead to avoid future confusion.

Reply:
The values here are used in parsing the value returned by the Phabricator API.
E.g.: `ReviewerStatus('accepted') == ReviewerStatus.ACCEPTED`

Reply:
I've appended enum-specific notes to the `render` module's docs.

I'm not sure that using a base class to make docs reference-able is a good call here for two reasons:

1. If a developer runs into one of these enums and isn't sure if the enum values are redundant, they can look up the enum usages to confirm their assumptions.
2. It adds structural complexity for the sole reason of making documentation more discoverable (the inheritance relationship doesn't provide any runtime/functional value). I'd rather add an inline comment to each enum pointing to the module docs, but even then, I'd rather that the developer confirm their assumption by checking usages.

Reply:
It's also worth noting that all of the data structures in `render.events` are all related to representing the data structures shared between this service and the Phabricator API endpoint. I've moved event-parsing logic to the module, so hopefuly that will also assist in making this more clear.